test/batching-test.js

```diff
@@ -38,7 +41,7 @@ class ComponentA extends React.Component {
     this.state = props.alt.stores.store.getState()
   }
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
We are changing here the hook in the React lifecycle. While this is likely ok, we need to be aware that if we encounter strange re-rendering of components this might be it.

Reply:
I guess this change is wrong..

Reply:
Assuming the intention of `componentWillMount` has not been abused, `componentDidMount` is the replacement recommendation 
https://reactjs.org/docs/react-component.html#unsafe_componentwillmount

The alternative would be to use `UNSAFE_componentWillMount`

Reply:
I just noticed this is used in `test/batching-test.js`, so definitely not a blocker for the PR. But, important to keep in mind when we are updating the UI

Reply:
OK. Thanks for the info.